script/scaffold/generate.py

```diff
@@ -116,14 +116,22 @@ def _custom_tasks(template, info) -> None:
             title=info.name,
             config={
                 "step": {
-                    "user": {"title": "Connect to the device", "data": {"host": "Host"}}
+                    "user": {
+                        "data": {
+                            "host": "[%key:common::config_flow::data::host%]",
+                            "username": "[%key:common::config_flow::data::username%]",
+                            "password": "[%key:common::config_flow::data::password%]",
+                        },
+                    }
                 },
                 "error": {
-                    "cannot_connect": "Failed to connect, please try again",
-                    "invalid_auth": "Invalid authentication",
-                    "unknown": "Unexpected error",
+                    "cannot_connect": "[%key:common::config_flow::abort::cannot_connect%]",
```

Comment:
was just looking through to start using theses new  common strings but i noticed that these reference "abort" but what i can see they should be sourced from "error" ?